ydb/public/api/protos/ydb_persqueue_v1.proto

```diff
@@ -38,7 +38,7 @@ message OffsetsRange {
 
 // In-session reauthentication and reauthorization, lets user increase session lifetime. You should wait for 'update_token_response' before sending next 'update_token_request'.
 message UpdateTokenRequest {
-    string token = 1;
+    string token = 1 [(Ydb.sensitive) = true];
```

Comment:
Fix logging of this in GrpcProxy

Reply:
Improved NYdbGrpc::FormatMessage using the TSecurityTextFormatPrinter